package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "soap",
-  "version": "0.28.0",
```

Comment:
Please revert this.  We bump the version when we publish.

Reply:
Sure. I have reverted the soap version change